core/drand_beacon.go

```diff
@@ -122,11 +122,13 @@ func (bp *BeaconProcess) Load() (bool, error) {
 
 	bp.log.Debugw("", "serving", bp.priv.Public.Address())
 
-	beaconID := metrics.UnknownBeaconID
 	if bp.group != nil {
-		beaconID = bp.group.ID
+		if bp.dkgDone {
+			metrics.DKGStateChange(metrics.DKGDone, bp.group.ID, false)
+		} else {
+			metrics.DKGStateChange(metrics.DKGNotStarted, bp.group.ID, false)
```

Comment:
You do the change to default in case of an empty group.ID below (line 150-154) but you don't do it here, why?

Reply:
Good call, fixed.